BetaCameras/ifm/O3D3xx.cs

```diff
@@ -326,6 +378,7 @@ protected override void ConnectImpl()
             string protocolVersion = _device.GetParameter("PcicProtocolVersion");
             if (_applicationId == -1)
             {
+                _cleanUpApplication = true;
                 for (int i = 1; i < 33; i++)
```

Comment:
Can you change this so that a new application is created if `_applicationId` is -1 and remove this loop that deletes all existing applications? (If too many existing applications already exist, throw an exception but do nothing. Basically, the camera object should *never* delete an application that it did not create and should always delete an application that it did create.